Tools/DumpRenderTree/cg/PixelDumpSupportCG.h

```diff
@@ -55,11 +46,7 @@ class BitmapContext : public RefCounted<BitmapContext> {
     ~BitmapContext()
     {
         if (m_buffer)
-#if PLATFORM(COCOA)
             free(m_buffer);
```

Comment:
Should probably change this to use `std::unique_ptr` to clean things up some day.

Reply:
I took a stab at doing so. I think the implementation is correct, but would appreciate your thoughts. I'll also wait for all platforms to run tests.